tree/30_generic_methods/file_check_same.cf

```diff
@@ -0,0 +1,63 @@
+#####################################################################################
+# Copyright 2013 Normation SAS
+# Copyright 2016 Janos Mattyasovszky
+#####################################################################################
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, Version 3.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+#####################################################################################
+
+# @name File check is same
+# @description Checks if two files are the same (hard links)
```

Comment:
I think this method name is a bit confusing. I originally understood it to mean "files have the same content" and was expecting to see the "diff" command in use.

Maybe we could rename "file_check_hard_links"?